### ISSUE TITLE
Can I use this library in deno? If so, how would I go about it?

### ISSUE THREAD

**Issue by Hugos68** [jhlywa/chess.js#379]
I'm building a chess application using chessjs on the client and need the same logic on my serverless function, the serverless function is running deno and i still want to use this package, is this possible?

**Comment by jhlywa** [comment]
Good question @Hugos68. I don't have any experience with any of the new JS runtimes ([deno](https://deno.land/) and [bun](https://bun.sh/)). Maybe you can give this a try and report your findings?

**Comment by Hugos68** [comment]
> Good question @Hugos68. I don't have any experience with any of the new JS runtimes ([deno](https://deno.land/) and [bun](https://bun.sh/)). Maybe you can give this a try and report your findings?
Alright, so I've successfully used this library with Deno. I used ``import { Chess } from "https://esm.sh/chess.js@1.0.0-beta.3";`` to test it and it works fine. One thing I noticed while testing was that when exporting to Pgn between the last header and ``*`` there wasn't a new line. This caused the reading of the export pgn to fail which seems rather strange to me. Not sure if this is related to the beta version or not. Besides the parsing I have yet to notice any problems running  Chessjs on deno, thanks for this amazing library!
